TicTacToeAppTests/ViewControllerSpec.swift

```diff
@@ -1,15 +1,51 @@
+
 import Quick
 import Nimble
 import TicTacToeApp
 
 class ViewControllerSpec: QuickSpec {
+
+  override func setUp() {
+    continueAfterFailure = false
+  }
+
   override func spec() {
+    
+    var controller: ViewController!
+    var infoLabel: UILabel!
+    
+    func tagButtonsInSequence(buttons: [UIButton]) {
+      var tag = 0
+        
+      for button in buttons {
+          button.tag = tag
+          tag += 1
+      }
+    }
+    
+    func makeMovesInSequence(controller: ViewController, buttonSequence: [Int]) {
+      for buttonIndex in buttonSequence {
+          controller.makeMove(controller.boardButtons[buttonIndex])
+      }
+    }
+
+    beforeEach {
+      controller = ViewController()
```

Comment:
Thanks for the explanation. Differing beforeEach() blocks might be a good idea once we get into grouping our tests better using describe blocks.